packages/logger/src/index.ts

```diff
@@ -1,32 +1,42 @@
-import * as debug from 'debug';
-import * as winston from 'winston';
+import debug from 'debug';
+import { format, Logger, createLogger, transports } from 'winston';
+import { TransformableInfo } from 'logform';
+import stringify from 'json-stringify-safe';
+import { EOL } from 'os';
 
 const { LOG_LEVEL = 'debug' } = process.env;
 
-const formatLogLevel = (level: string): string => {
-  // Clean [31m and invisible character used to show colored strings by Winston
-  const cleaned = level.replace(/\[\d{2}m{1}|[^a-zA-Z ]/g, '');
-  switch (cleaned) {
-    case 'info':
-      return `🍺 ${level}`;
-    case 'warn':
-      return `❗️ ${level}`;
-    case 'error':
-      return `🔥 ${level}`;
-    default:
-      return `🤷‍♂️ ${level}`;
-  }
+const LevelEmoji: Record<string, string> = {
+  info: '🍺',
+  warn: '❗️',
+  error: '🔥',
+  default: '🤷‍♂️',
 };
 
-const instance: winston.Logger = winston.createLogger({
+// have to cast to any to access additional params value
+const splatIndex = Symbol.for('splat') as any;
```

Comment:
I see 2 possible ways of making this a bit more readable for people who didn't have to go through forums or Winston docs and still make this readable.
1. Add a JSDoc right above explaining what it does, why are you looking for that specific one. `// Winston adds all parameters in their 'splat' property...`
2. Make this into a function. 

```
/**
* Returns all available Winston input values.
* @param {object} info - Winston information when logging gets used.
*/
const getWinstonInputValues = <T>(info: TransformableInfo):T => Symbol.for('splat') as T;
```

Potentially even add this logic in the same function and rename it to `getWinstonParameters`.  
```
return paramsIn instanceof Array ? paramsIn : [paramsIn]).map((v: unknown) => EOL + stringify(v);

```